shared/src/main/scala/squants/time/Frequency.scala

```diff
@@ -26,6 +30,36 @@ final class Frequency private (val value: Double, val unit: FrequencyUnit)
   protected[squants] def timeIntegrated = Each(toHertz)
   protected[squants] def time = Seconds(1)
 
+  // Below are two failed attempts to invert the TimeIntegral * Frequency method to support Frequency * TimeIntegral
```

Comment:
I think we should remove the comments from lines 33 - 44 into the PR. I'd like to preserve them, but I don't think they need to be in the source.